.github/workflows/ci.yml

```diff
@@ -62,37 +63,39 @@ jobs:
 
         - name: Set up QEMU
           uses: docker/setup-qemu-action@v2
+          with:
+            platforms: arm64
 
         - name: Set up Docker Buildx
           uses: docker/setup-buildx-action@v2
 
         - name: Log in to the Container registry
           uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
-          if: github.ref == 'refs/heads/main'
           with:
             username: ${{ secrets.DOCKER_HUB_USER }}
             password: ${{ secrets.DOCKER_HUB_PASS }}
 
         - name: Log in to the Container registry
           uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
-          if: github.ref == 'refs/heads/main'
           with:
             registry: ${{ env.GITHUB_REGISTRY }}
             username: ${{ github.actor }}
             password: ${{ secrets.GITHUB_TOKEN }}
 
-        - name: Build and Push
+        - name: Build and Push Container
           uses: docker/build-push-action@v4
           with:
-            context: backend
+            context: frontend
             platforms: linux/amd64,linux/arm64
-            push: ${{ github.ref == 'refs/heads/main' }}
+            push: true
```

Comment:
I think this can be omitted

Reply:
no, default is false. see https://github.com/docker/build-push-action#inputs